src/modules/core-milestone/core-milestone.tsx

```diff
@@ -0,0 +1,289 @@
+margin-horizontal: ${minorScale(1)};
+margin-left:0px;
+`;
+
+const StyledBaptisedWrapper = Styled.View`
+width: ${minorScale(3)};
+height: ${minorScale(3)};
+margin-horizontal: ${minorScale(1)};
+margin-left:${minorScale(1)}px;
+`;
+
+const StyledSmallGroupWrapper = Styled.View`
+width: ${minorScale(3)};
+height: ${minorScale(3)};
+margin-horizontal: ${minorScale(1)};
+margin-left:${minorScale(1)}px;
+`;
+
+const StyledInMinistryWrapper = Styled.View`
+width: ${minorScale(3)};
+height: ${minorScale(3)};
+margin-horizontal: ${minorScale(1)};
+margin-left:${minorScale(1)}px;
+`;
+
+const StyledActiveMissionWrapper = Styled.View`
+width: ${minorScale(3)};
+height: ${minorScale(3)};
+margin-horizontal: ${minorScale(1)};
+margin-left:${minorScale(1)}px;
+`;
+
+export const CoreMilestone: React.FC<IProps> = ({
+  hasAttendedClass101,
+  hasAttendedClass201,
+  hasAttendedClass301,
+  hasAttendedClass401,
+  hasSignedMembershipAgreement,
+  hasSignedMaturityCovenant,
+  hasSignedMinistryCovenant,
+  hasSignedMissionCovenant,
+  hasAcceptedChrist,
+  isBaptised,
+  isInSmallGroup,
+  isInMinistry,
+  isActiveInMissions,
+  isWhite,
+  isAdult,
+  gender,
+  isStudent,
+  isChild,
+}): React.ReactElement => {
+  const renderClassMileStoneIcon = () => {
+    /**
+     * class milestone
+     */
+    return (
+      <StyledClassMilestoneWrapper isWhite={isWhite} testID="class-milestone">
+        <StyledLeftHasAttendedClass301Wrapper
+          hasAttendedClass301={hasAttendedClass301}
+          hasSignedMinistryCovenant={hasSignedMinistryCovenant}
+          colorMilestone={getColorbaseOnUserType()}
+          isWhite={isWhite}
+        />
+        <StyledEmptyViewWrapper>
+          <StyledClasshasAttendedClass201Wrapper
+            hasAttendedClass201={hasAttendedClass201}
+            hasSignedMaturityCovenant={hasSignedMaturityCovenant}
+            colorMilestone={getColorbaseOnUserType()}
+            isWhite={isWhite}
+          />
+          <StyledClassHasAttendedClass401Wrapper
+            hasAttendedClass401={hasAttendedClass401}
+            hasSignedMissionCovenant={hasSignedMissionCovenant}
+            colorMilestone={getColorbaseOnUserType()}
+            isWhite={isWhite}
+          />
+        </StyledEmptyViewWrapper>
+        <StyledClassHasAttendedClass101Wrapper
+          hasAttendedClass101={hasAttendedClass101}
+          hasSignedMembershipAgreement={hasSignedMembershipAgreement}
+          colorMilestone={getColorbaseOnUserType()}
+          isWhite={isWhite}
+        />
+      </StyledClassMilestoneWrapper>
+    );
+  };
+
+  const getColorbaseOnUserType = () => {
+    if (isAdult === true) {
+      if (gender === 'M') {
+        return '#3A8E5D';
+      } else if (gender === 'F') {
+        return '#0290B7';
+      }
+    } else if (isStudent === true) {
+      return '#CB6342';
+    } else if (isChild === true) {
+      return '#5AC9F5';
+    } else {
+      return '#56C4C4';
+    }
+    return '#56C4C4';
```

Comment:
This can simply be `if (isAdult)` instead of `if (isAdult === true)`. Same for the other `=== true` conditions.

Reply:
I have fixed it.